lib/tocurrency.js

```diff
@@ -0,0 +1,6 @@
+function toCurrency (value) {
+  const decimalString = value ? value.replace(/[^\d|.]+/g, '') : ''
+  return decimalString === '' ? null : Math.round((Number(decimalString) + Number.EPSILON) * 100) / 100
+}
```

Comment:
Converting currencies to floats fills me with horror - I assume that the use of `Number.EPSILON` is intended to deal with this, but I wonder if it would be better to use something that explicitly handles currency?

Reply:
Replaced bespoke code with [currency.js](https://github.com/scurker/currency.js) library.